.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+​
+- [ ] Changelog entry have been added / updated
+- [ ] Docs/Stories have been added / updated (for bug fixes / features)
+​
+​
+## PR Type
+What kind of change does this PR introduce?
+​
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
+[ ] Feature
+[ ] Code style update (formatting, local variables)
+[ ] Refactoring (no functional changes, no api changes)
+[ ] Build related changes
+[ ] CI related changes
+[ ] Other... Please describe:
```

Comment:
should we also add documentation and stories to this list?

Reply:
this option is also in the requirements above :+1:

Reply:
Sorry, how do you mean that?

Reply:
when I change something in the docs/stories then I will check this above in the requirements.
Which change happend will be checked here in the type, for example when I fix something in a story then I'm using `Docs/Stories have been added / updated (for bug fixes / features)` and `Bugfix`

Reply:
And new docs are a `Feature`?

Reply:
yes :+1:

Reply:
That doesn't make much sense to me :)

But we should perhaps first talk about the purpose of this "PR Type" section and then decide about the types.

Reply:
I also think there should be a `Docs/Stories` type. Since they do not add any new functionality, the Docs are no feature in my opinion.

Reply:
Instead of a PR type, we could also add more labels and label each PR. This should then be added as a bullet point to the `PR Checklist` section (otherwise everyone will forget about the labels)